addons/website_sale/static/src/js/website_sale_tour_buy.js

```diff
@@ -29,9 +29,12 @@ tour.register('shop_buy_product', { test: true, url: '/shop' },
             trigger: '#product_detail form[action^="/shop/cart/update"] .btn',
         },
         {
+            content: "wait to add suggested because website_sale_options change tour",
```

Comment:
A bug in the new tour, I look to fix because without fix we need to add stupid step